pandas/core/strings.py

```diff
@@ -505,6 +507,280 @@ def f(x):
     return result, name
 
 
+def str_extract_frame(arr, pat, flags=0):
+    """
+    For each subject string in the Series, extract groups from the
+    first match of regular expression pat. This function is called from
+    str_extract(expand=True), and always returns a DataFrame.
+
+    .. versionadded:: 0.13.0
+
+    Parameters
+    ----------
+    pat : string
+        Regular expression pattern with capturing groups
+    flags : int, default 0 (no flags)
+        re module flags, e.g. re.IGNORECASE
+
+    Returns
+    -------
+    DataFrame with one row for each subject string, and one column for
+    each group. Any capture group names in regular expression pat will
+    be used for column names; otherwise capture group numbers will be
+    used. The dtype of each result column is always object, even when
+    no match is found.
+
+    Examples
+    --------
```

Comment:
Are the examples needed here? Because it seems duplicate from the `str_extract` docstring, at that one is the only that users will see?